### DISCUSSION TITLE
How do I make sure my friend doesn't see my IP address

### DISCUSSION THREAD

**Discussion by MoenyOfficial** [community/community#133266] [New to GitHub]
![Schermafbeelding 2024-07-23 152415](https://github.com/user-attachments/assets/d4cee2d5-d427-4716-98de-2a95bf0480e5)

Can I disable this option? will he see my IP? Is this only for Github? 

I wanna know because I don't want stuff to happen

**Comment by ghostinhershell** [comment]
Hi @MoenyOfficial, 

Thanks so much for reaching out in our community! It looks like this question was answered in this discussion here: https://github.com/orgs/community/discussions/123172. Take a look and let me know if you have any additional questions on this topic. Thanks!
